Server/XCUIApplicationStateTypedef.h

```diff
@@ -2,7 +2,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
-#ifndef XCT_UI_TESTING_AVAILABLE
```

Comment:
@jmoody what do you think about it?

Reply:
It looks like Apple removed this enum. I cannot find it in the private headers.

I recommend you open an XCUITest project in Xcode 11.5 (the Palisade project would be a good choice) - and see if symbols like `XCUIApplicationStateRunning` are available.

If Apple has removed this, then we will need to make changes to DeviceAgent because values are used in a few places - notably in Application.m

Reply:
We've investigated the issue with the PR in the current state and didn't encounter any particular issues with Enums being deleted from private headers. This whole situation with the Enum is strange because the fact that it's deleted or is due to be deleted can't be found in any changelogs and documentation.

We're using the copy of it in the project, so there shouldn't be any issues.

Regards to the macro `XCT_UI_TESTING_AVAILABLE`: it is no longer used in the project, so we shouldn't encounter this issue again.

Reply:
We've investigated the issue with the PR in the current state and didn't encounter any particular issues with Enums being deleted from private headers. This whole situation with the Enum is strange because the fact that it's deleted or is due to be deleted can't be found in any changelogs and documentation.

We're using the copy of it in the project, so there shouldn't be any issues.

Regards to the macro `XCT_UI_TESTING_AVAILABLE`: it is no longer used in the project, so we shouldn't encounter this issue again.